examples/bootc-uki/build.base

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -eux
+
+cd "${0%/*}"
+
+cargo build --release --features=pre-6.15 --bin cfsctl --bin composefs-setup-root
+
+cp ../../target/release/cfsctl .
```

Comment:
Should use [bootc internals cfs](https://github.com/bootc-dev/bootc/pull/1477) instead